### PR TITLE
test: fix enzyme snaps

### PR DESCRIPTION
**Pull request by craigbilner**
In light of Enzyme's poor support of RN snapshotting, this PR removes enzyme from native tests and refactors some packages that were not conforming to the new test patterns

### CODE REVIEW

File: packages/card/__tests__/shared.web.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import { mount } from "enzyme";
+import React from "react";
+import { Text } from "react-native";
+import { shallow, mount } from "enzyme";
```
Comment (colincclark) [inline packages/card/__tests__/shared.web.js]: is `mount` used here? did this pass linting?

File: packages/image/__tests__/android/__snapshots__/image-with-style.android.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+                      "right": 0,
+                      "top": 0,
+                      "width": "100%",
+                    }
+                  }
+                />
+                <View
+                  style={
+                    Object {
+                      "flex": 1,
```
Comment (colincclark) [inline packages/image/__tests__/android/__snapshots__/image-with-style.android.test.js.snap]: thought these were `flex: 1` by default?

File: packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap [hunk 3]
```diff
@@ -1,122 +1,213 @@
...
+    <View>
+      <Gradient
+        degrees={264}
+      >
+        <View>
+          <Svg
+            height={164.46700507614213}
+            version="1.1"
+            viewBox="145 50 108 120"
+            width={164.46700507614213}
```
Comment (colincclark) [inline packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap]: weird width

File: packages/image/__tests__/shared.native.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { ImageBackground, Modal } from "react-native";
+import Link from "@times-components/link";
+import TestRenderer from "react-test-renderer";
```
Comment (colincclark) [inline packages/image/__tests__/shared.native.js]: ordering of imports. would be good to get the lint rule going on this, especially as the props ordering rule has proved itself now

File: packages/card/__tests__/shared.web.js (hunk 1 above)
Comment (craigbilner) [reply packages/card/__tests__/shared.web.js]: yes, we use mount for the shared tests, it's passed in at the bottom

File: packages/image/__tests__/android/__snapshots__/image-with-style.android.test.js.snap (hunk 2 above)
Comment (craigbilner) [reply packages/image/__tests__/android/__snapshots__/image-with-style.android.test.js.snap]: will review after :+1:

File: packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap (hunk 3 above)
Comment (craigbilner) [reply packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap]: just testing the calculation

File: packages/image/__tests__/shared.native.js (hunk 4 above)
Comment (craigbilner) [reply packages/image/__tests__/shared.native.js]: :speaking_head: Monday tech chat

File: packages/article-topics/__tests__/android/__snapshots__/article-topics.android.test.js.snap [hunk 5]
```diff
@@ -0,0 +1,90 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`1. group of topics 1`] = `
+<View>
```
Comment (colincclark) [inline packages/article-topics/__tests__/android/__snapshots__/article-topics.android.test.js.snap]: I think these ones will fail jest-linting on `MaxGenericElementWarning`

File: packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap [hunk 6]
```diff
@@ -1,122 +1,213 @@
...
+    >
+      <View>
+        <Image
+          source={
+            Object {
+              "uri": "http://example.com/image.jpg?resize=1440",
+            }
+          }
+        />
+        <View>
```
Comment (colincclark) [inline packages/image/__tests__/ios/__snapshots__/image.ios.test.js.snap]: this one fails jest-lint on `MaxGenericElementWarning`

File: packages/article-topics/__tests__/android/__snapshots__/article-topics.android.test.js.snap (hunk 5 above)
Comment (craigbilner) [reply packages/article-topics/__tests__/android/__snapshots__/article-topics.android.test.js.snap]: :speaking_head: will address separately

**Pull request merged**